Formula/arpack.rb

```diff
@@ -38,9 +36,7 @@ def install
     pkgshare.install "TESTS/testA.mtx", "TESTS/dnsimp.f",
                      "TESTS/mmio.f", "TESTS/debug.h"
 
-    if build.with? "mpi"
-      (libexec/"bin").install (buildpath/"PARPACK/EXAMPLES/MPI").children
-    end
+    (libexec/"bin").install (buildpath/"PARPACK/EXAMPLES/MPI").children if build.with? "mpi"
```

Comment:
The one above is fine, but this one shoots over 80 characters quite a bit, which may be why it was turned into a block.

Reply:
The problem with the block is the `brew audit` warning. I tend to like the one-liner because it has no warnings.

Reply:
Hmm. IMO it shouldn't be encouraging situations where that advice causes the line length to shoot more than 5% over the ideal 80 characters width. CC @Ilovezfs for thoughts.

Reply:
I mean we could cheat & do something like:
```ruby
dir = buildpath/"PARPACK/EXAMPLES/MPI"
(libexec/"bin").install dir.children if build.with? "mpi"
```
But that seems mildly silly.

Reply:
Current versions of rubocop will automatically flag these and if the change causes the line to go passed the eightieth column, then rubocop will not flag them. So if this was already passing brew style then there's nothing that needs to be changed.

Reply:
Ah, you're correct, and I need to learn how to read apparently. Sorry! :see_no_evil:

```
== Library/Taps/homebrew/homebrew-core/Formula/arpack.rb ==
C: 27:  5: Favor modifier if usage when having a single-line body. Another good alternative is the usage of control flow &&/||.

1 file inspected, 1 offense detected
```

Reply:
So if any of these changes cause the line to exceed the eightieth column then we should not make that particular change, and if any of the changes don't make the line exceed the eightieth column then we should make that particular change.

Reply:
OK, now I'm not on my phone. So I can see we should make the change at line 27 but not make the change at line 39.

Reply:
Indeed. I'm putting it back inside a block.